sections/data_release_production_pipelines.tex

```diff
@@ -71,26 +75,332 @@ \section{Data Release Production}
+\begin{itemize}
+\item The PSF and WCS must have a consistent definition of object centroids.  Celestial positions from a reference catalog are transformed via the WCS to set the positions of stars used to build the PSF model, but the PSF model is then used to measure debiased centroids that feed the WCS fitting.
+\item The later stages of photometric calibration and PSF modeling require secure star selection and colors to infer their SEDs.  Magnitude and morphological measurements from ImChar stages are aggregated the reference catalog in the subsequent JointCal stage, allowing these colors and classifications to be used for PSF modeling in the following ImChar stage.
+\end{itemize}
+
+The ImChar and JointCal iteration is itself interleaved with background matching, described in section~\ref{sec:drp_coaddition_and_diffim}.  This allows the best backgrounds and masks to be defined in the \hyperref[sec:drpBackgroundMatchAndReject]{BackgroundMatchAndReject} before the final Source measurements, image characterizations, and calibrations.
+
+Each ImChar pipeline runs on a single visit, and each JointCal pipeline runs simultaneously on all visits within a single tract, allowing tracts to be run entirely independently.
+
+The final output data products of the ImChar/JointCal iteration are the Source table and CalExp (calibrated exposure) images.  The latter includes Image, Mask, Variance, Background, PSF, WCS, Calib components that we will track separately.
+
+There are also several intermediate versions of the Source and CalExp data products passed between the ImChar/JointCal pipelines, as well as...
+
+\begin{note}[TODO]
+  finish discussing data products.
+\end{note}
+
 \subsubsection{BootstrapImChar}
 \label{sec:drpBootstrapImChar}
-\subsubsection{BootstrapJointCal}
-\label{sec:drpBootstrapJointCal}
+
+The BootstrapImChar pipeline is the first thing run on each science exposure in a data release.  It has the difficult task of bootstrapping multiple quantities (PSF, WCS, photometric calibration, background model, etc.) that each normally require all of the others to be specified when one is fit.  As a result, while the algorithmic components to be run in this pipeline are generally clear, their ordering and specific requirements are not; algorithms that are run early will have a harder task than algorithms that are run later, and some iteration will almost certainly be necessary.
+
+A plausible (but by no means certain) high-level algorithm for this pipeline is given below in pseudocode.  Highlighted terms are described in more detail below the pseudocode block.
```

Comment:
A trivial point, but -- although this is now building on Travis, neither the Travis generated PDF nor one I produced on my laptop have any noticeable highlighting in this block.